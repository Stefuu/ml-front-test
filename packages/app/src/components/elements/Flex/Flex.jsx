import { withProps } from 'recompose'
import Box from '../Box'

const Flex = withProps(props => ({ display: 'flex', ...props }))(Box)

export default Flex
