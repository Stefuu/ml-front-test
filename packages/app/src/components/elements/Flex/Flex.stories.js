import { storiesOf } from '@storybook/react'
import { doc } from 'storybook-readme'
import readmeFile from './README.md'

storiesOf('app/components/elements/Flex', module)
  .addParameters({ jest: ['Flex.test.js'] })
  .add('README', doc(readmeFile))
