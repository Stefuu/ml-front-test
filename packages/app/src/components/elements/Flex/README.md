# Flex

## Extended Box with display flex
