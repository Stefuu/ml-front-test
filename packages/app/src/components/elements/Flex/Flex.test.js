import React from 'react'
import { render } from 'react-testing-library'
import Flex from './Flex'

describe('Flex', () => {
  it('Deve renderizar [snapshot]', async () => {
    const { container } = render(<Flex />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
