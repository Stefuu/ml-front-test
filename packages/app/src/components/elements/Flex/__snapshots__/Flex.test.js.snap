// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex Should render [snapshot] 1`] = `
<div
  class="sc-bdVaJa cyMNyE"
/>
`;
